website/src/components/quickstartTOC/index.js

```diff
@@ -0,0 +1,223 @@
+// This component is used to build the functionality of the quickstart guides
+// Each H2 (##) in the markdown file is a step in the guide
+
+import React, { useState, useEffect } from "react";
+import clsx from "clsx";
+import style from "./styles.module.css";
+import { useLocation, useHistory } from "@docusaurus/router";
+import queryString from "query-string";
+
+function QuickstartTOC() {
+  const history = useHistory();
+  const location = useLocation()
+  const locationPath = useLocation().pathname;
+  const queryParams = queryString.parse(location.search);
+
+  const activeStepLocal = typeof localStorage !== "undefined" ? localStorage.getItem(locationPath) : 1;
+  const activeStepParam = queryParams.step ? (queryParams.step) : activeStepLocal ? activeStepLocal : 1;
+
+  const [mounted, setMounted] = useState(false);
+  const [tocData, setTocData] = useState([]);
+  const [activeStep, setActiveStep] = useState(activeStepLocal || 1);
+  const [activeQuickstart, setActiveQuickstart] = useState(locationPath);
```

Comment:
eslint stating `'setActiveQuickstart' is assigned a value but never used`. Is it possible to remove this state and use `locationPath` directly, or do we anticipate the `activeQuickstart` state changing down the road?

Reply:
Good call. I don't anticipate needing that state down the road. Just nuked it in favor of `locationPath`.